Gruntfile.js

```diff
@@ -435,6 +444,13 @@ module.exports = function (grunt) {
             };
         });
 
+        // Copy any LESS/CSS files from extensions to the less task file list.
+        extensions.forEach(function(extension) {
```

Comment:
I think as a follow-up, one optimization we could do is to rely on `fs.find` to list all the files with `.less` for each extension and generate the dist css file based on that path instead of manually storing it in the `bramble-extensions.json` file.